src/libs/ValidationUtils.js

```diff
@@ -69,6 +69,15 @@ function isValidAge(date) {
     return moment().diff(moment(date), 'years') >= 18;
 }
 
+/**
+ *
+ * @param {String} phoneNumber
+ * @returns {Boolean}
+ */
+function isValidPhoneWithSpecialChars(phoneNumber) {
+    return CONST.REGEX.PHONE_WITH_SPECIAL_CHARS.test(phoneNumber);
```

Comment:
This doesn't test for max length. I know we do that on the input, but  I worry that this will be used without that.